src/librustc_mir/interpret/eval_context.rs

```diff
@@ -651,7 +688,11 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         self.frame_mut().locals = locals;
 
         M::after_stack_push(self)?;
-        info!("ENTERING({}) {}", self.frame_idx(), self.frame().instance);
+
+        let span = info_span!("frame", "{}", instance);
+        self.frame_mut().tracing_span.enter(span);
```

Comment:
Is there some way we can avoid rightwards drift with stacks of 50 functions and more?

Looks like the indentation is done fully automatically by tracing so this might be hard to control.

Reply:
There is: https://github.com/davidbarsky/tracing-tree/pull/15. This is in the most recent release of tracing-tree on crates.io.

Reply:
Oh that's awesome. :)

Reply:
this is already avoided :) after a depth of 10 we wrap around. Example: 
https://user-images.githubusercontent.com/2067774/89716739-8c86b800-d97d-11ea-9b74-78f040b845aa.png

Reply:
Hm, could there be some indication like a "..." to show that wrap-around happened?